mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -72,15 +116,17 @@ DIGITS_AFTER_DECIMAL = 2
 
 tallyWeights = (tbody, hash) ->
```

Comment:
This function is still too long. Please break it into multiple functions.  You should be able to make at least 2 (maybe more) functions that can be tested in Jest.  You do this by sending values (rather than DOM references) as objects.